client/src/ui/Form.jsx

```diff
@@ -326,6 +351,96 @@ export default class Form extends Component {
     }
   }
 
+  getGradeOptions() {
+    this.gradeReceived = ["A+", "A", "A-", "B+", "B", "B-", "C+", "C", "C-", "D+", "D", "D-", "F"];
+    return this.gradeReceived.map((grade) =>
+      ({
+        "value": grade,
+        "label": grade
+      }))
+  }
+
+getMajorOptions() {
+    this.majorsOptions = ["Computer science",
```

Comment:
Lets put these in another file and import from there. I see that this list is repeated later on in code too. We dont want that if possible.

Reply:
Should I delete Form.jsx where this is duplicated?

Reply:
I think it might be a good idea to delete `Form.jsx` and other unused files in another PR